modules/airavata-helix/task-core/src/main/java/org/apache/airavata/helix/core/AbstractTask.java

```diff
@@ -60,11 +61,15 @@
 
     private TaskCallbackContext callbackContext;
     private TaskHelper taskHelper;
+    private HelixParticipant participant;
 
     private int retryCount = 3;
 
     @Override
     public void init(HelixManager manager, String workflowName, String jobName, String taskName) {
+        if (participant != null) {
+            participant.registerRunningTask(this);
```

Comment:
Print a warn log mentioning that registration will not happen if the participant is null